pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: collection
-version: 1.16.0
+version: 1.17.0-dev
```

Comment:
We are still on `1.16.0`.

https://github.com/dart-lang/collection/pull/211

Reply:
ACK.